### PR TITLE
Use correct stackName when specified in provider closes #14

### LINKED ISSUE #14: Incorrect stackName when specified from provider
Hello,
we noticed that when we have an overriden stackName specified from the provider.stackName property this plugin doesn't honor it and uses the default one (service + "-" + stage). Which results in an incorrect stackname being used

For example when using custom variables

```
provider:
  name: aws
  stackName: stack-${self:custom.${opt:stage}.suffix}

custom:
  dev:
    suffix: example
  prod:
    suffix: example
```

We have a PR with a fix incoming

### PR DESCRIPTION
**Pull request by cbm-gplassard**

### CODE REVIEW

File: index.ts [hunk 1]
```diff
@@ -29,7 +29,7 @@ class ServerlessCloudWatchLogsTagPlugin {
   }
 
   private get stackName(): string {
-    return `${this.serverless.service['service']}-${this.options.stage}`;
+    return this.awsService.naming.getStackName();
```
Comment (clementheliou) [inline index.ts:32]: I guess we can do a little bit shorter...
```suggestion
    return this.awsService.getStackName();
```
Didn't test this suggestion : it's only based on type found in `@types/serverless/plugins/aws/provider/awsProvider.d.ts`:
```typescript
declare class Aws {
    constructor(serverless: Serverless, options: Serverless.Options)

    naming: { [key: string]: () => string };
    getProviderName(): string;
    getRegion(): string;
    getServerlessDeploymentBucketName(): string;
    getStage(): string;
}
```

**Review by clementheliou — COMMENTED** [review]
Thank you for providing us with this PR. 👍 
That's a great help for us.

I suggested a little change to you and will be able to approve your PR short after.

File: index.ts (hunk 1 above)
Comment (cbm-gplassard) [reply index.ts:32]: I don't think it would work, there isn't a getStackName() in the AWS class (but I can test tomorrow)

Comment (clementheliou) [reply index.ts:32]: Fine. Please keep me posted with the result, I'll approve it as soon as received.

Comment (cbm-gplassard) [reply index.ts:32]: Hello again, so I tested with the 'naming' and got the following (truncated) log

```
Serverless: Tagged LogGroup /aws/lambda/... with the following tags: ...
```

Without the naming I get a

```
TypeError: this.awsService.getStackName is not a function
```

**Review by clementheliou — APPROVED** [review]

**Comment by cbm-gplassard** [comment]
Thanks for the merge !

**Comment by clementheliou** [comment]
> Thanks for the merge !

You're welcome.
New version published => `1.1.0`.

**Pull request merged**